docs/source/usage/plugins/saxs.rst

```diff
@@ -0,0 +1,119 @@
+    For that reason, the envelopes and the offset can't be set in the ``.param`` file yet.
+    The probe is always a plane wave.
+    Beam rotation works.
+
+The alignment settings define a beam coordinate system with :math:`\hat{z}  = \hat{k}` and :math:`\hat{x}`, :math:`\hat{y}` perpendicular to the x-ray propagation direction.
+It is always a right-hand system oriented in such way that for propagation parallel to the PIC x- or y-axis :math:`\hat{x}_{\text{beam}} = - \hat{z}_{\text{PIC}}` and if :math:`{\bf k }` is parallel to  the PIC z-axis  :math:`\hat{x}_{\text{beam}} = - \hat{y}_{\text{PIC}}`. The orientation can be then adjusted with the `RotationParam` setting.
+
+.. TODO: Add other parameters after the coordinate transform has been fixed and the settings moved back to the .param file.
+
+================   ===============================================================================================================================
+  Setting                      Description 
+================   ===============================================================================================================================
+``ProbingSide``    The side from which the x-ray is propagated.
+                   Set `X`, `Y` or `Z` for propagation along one of the PIC coordinate system axes;
+                   `XR`, `YR` or `ZR` for propagation against the axis.
+
+``RotationParam``  Rotation of the beam axis, :math:`z_{\text{beam}}`, from the default orientation ( perpendicular the the simulation box side ).
+                   Set the beam yaw and pitch angles in radians.
+================   ===============================================================================================================================
+
+.. TODO: Add BEAM_OFFSET in between after the coordinate transform has been fixed.
+
+The coordinate transfer from the PIC system to the beam system is performed in the following order:
+rotation to one of the default orientations (``ProbingSide`` setting), additional rotation (``RotationParam`` ). This has to be taken into account when defining the experimental setup.
+
+
+.cfg file
+^^^^^^^^^
+
+For a specific (charged) species ``<species>`` e.g. ``e``, the scattering can be computed by the following commands.
+
+========================================= ============================================================================================================================================
+Command line option                       Description
+========================================= ============================================================================================================================================
+``--<species>_saxs.period``               Period at which the plugin is enabled (PIC period syntax). Only the intensity from this steps is accumulated.
+                                          Default is `0`, which means that the scattering intensity in never calculated and therefor off.
```

Comment:
Well yes, exactly @PrometheusPi this is the integration step. But since it is the usual PIC plugin period option it can be used to enable the integration at chosen simulation stages only. With the coordinate transform working and a custom temporal profile, this could be used for simulating single X-ray pulses at different staegs.